core/src/scene/sceneLoader.cpp

```diff
@@ -94,21 +94,12 @@ bool SceneLoader::applyUpdates(const std::shared_ptr<Platform>& platform, Scene&
         }
 
         if (value) {
-            try {
-                // Dummy node to trigger YAML exception on YAML syntax errors
-                auto parse = YAML::Load(update.path);
```

Comment:
This check isn't useful - the coded path for a yaml node need not be valid yaml. It might be useful to check whether the individual tokens in the path are valid yaml, but this doesn't check that.